keploy/keploy.go

```diff
@@ -493,32 +536,61 @@ func (k *Keploy) check(runId string, tc models.TestCase) bool {
 	}
 
 	// test application reponse
-	r, err := http.NewRequest("POST", k.cfg.Server.URL+"/regression/test", bytes.NewBuffer(bin))
-	if err != nil {
-		k.Log.Error("failed to create test request request server", zap.String("id", tc.ID), zap.String("url", tc.URI), zap.Error(err))
-		return false
-	}
-	k.setKey(r)
-	r.Header.Set("Content-Type", "application/json")
+	if k.cfg.Server.GrpcEnabled {
+		r, err := k.grpcClient.Test(k.Ctx, &proto.TestReq{
```

Comment:
Since, there are HTTP and GRPC_EXPORT types of tcs. Therefore, it should have proto.TestReq.GrpcResp field when tcs type is GRPC_EXPORT.

Reply:
Done